python/cuda_parallel/cuda/parallel/experimental/_iterators.py

```diff
@@ -433,6 +417,25 @@ def transform_dereference(it_state_ptr):
         # ATTENTION: op_caller here (see issue #3064)
         return op_caller(source_dereference(it_state_ptr))
 
+    prefix = f"transform_{it.prefix}_{op.__name__}"
+    ltoirs =  [
+        *it.ltoirs,
+        _ncc(
+            f"{prefix}_advance",
+            transform_advance,
+            numba.types.void(
+                numba.types.CPointer(numba.types.char), _DISTANCE_NUMBA_TYPE
+            ),
+        )[0],
+        _ncc(
+            f"{prefix}_dereference",
+            transform_dereference,
+            op_return_ntype(numba.types.CPointer(numba.types.char)),
+        )[0],
+        # ATTENTION: NOT op_caller here! (see issue #3064)
+        op_ltoir
```

Comment:
important: `op_ltoir` is a `[ltoir, ret_type]` now:
```suggestion
        op_ltoir[0]
```

Reply:
I fixed this by assigning `op_ltoir` to just the LTOIR.